src/app/code/community/Shopgate/Cloudapi/Helper/Api2/Quote.php

```diff
@@ -25,6 +25,7 @@ class Shopgate_Cloudapi_Helper_Api2_Quote extends Mage_Core_Helper_Abstract
     const KEY_ITEMS       = 'items';
     const KEY_TOTALS      = 'totals';
     const KEY_ITEM_ERRORS = 'errors';
+    const KEY_CART_PRICE_DISPLAY_SETTINGS = 'cart_price_display_settings';
```

Comment:
Should be aligned with the constants above

Reply:
Done